packages/cli/src/oclif/buildFlags.js

```diff
@@ -1,25 +1,24 @@
-const { flags } = require('@oclif/command');
+const { Flags } = require('@oclif/core');
 const { pickBy } = require('lodash');
 
 const { formatStyles } = require('../utils/display');
 
 const baseFlags = {
-  format: flags.string({
+  debug: Flags.boolean({
+    char: 'd',
+    description: 'Show extra debugging output.',
+    // pull from env?
+  }),
+  format: Flags.string({
     char: 'f',
     options: Object.keys(formatStyles),
     default: 'table',
```

Comment:
Curious if [this change](https://github.com/oclif/core/blob/main/guides/PRE_CORE_MIGRATION.md#update-default-property-on-flag-definitions) around `default` build `async` now applies to even these literal values? 🤔

Reply:
Good question! It digs into the oclif package but it ends up [here](https://github.com/oclif/core/blob/main/src/interfaces/parser.ts#L63). In essence, type `T` there for `Flags.string` is `string` which is the default type for `default`.